std/zip.d

```diff
@@ -302,9 +323,6 @@ final class ZipArchive
     private uint _totalEntries;
     private bool _isZip64;
     static const ushort zip64ExtractVersion = 45;
-    static const int digiSignLength = 6;
-    static const int eocd64LocLength = 20;
-    static const int eocd64Length = 56;
```

Comment:
These constants were public, so I think they would need to go through a deprecation cycle.

Reply:
Is I wrote in my comment above, I felt a little bit unsure if they should be deprecated, but indeed, it should. ;-)